e2e/etcd_test.go

```diff
@@ -430,16 +429,33 @@ func spawnWithExpects(args []string, xs ...string) error {
 	if err != nil {
 		return err
 	}
+	// process until either stdout or stderr contains
+	// the expected string
+	var (
+		lines    []string
+		lineFunc = func(txt string) bool {
+			return len(txt) > 0
```

Comment:
`lineFunc := func(string) { return true}`? There should never be a blank line because of `https://github.com/coreos/etcd/blob/master/pkg/expect/expect.go#L67`